samples/openthread/cli/boards/nrf54l15dk_nrf54l15_cpuapp_ns.conf

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2024 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# Disable the unsupported driver
+CONFIG_NRFX_UARTE0=n
+
+# Workaround required as Zephyr L2 implies usage of NVS backend for settings.
+# It should be removed once the proper fix will be applied in Zephyr.
```

Comment:
Is this ongoing?

Reply:
switching to ZMS has been added to nrf54l15 target two days ago so I assumed that it is https://github.com/nrfconnect/sdk-nrf/commit/3ad9997ca9d769a0c37bdd03abaea90cb512b91f
@kkasperczyk-no this should be selected for TF-M as well, right?

Reply:
I mean, i know ZMS should be used always for 54L (both ns an s) but do we need to enable it in all samples? I was more askig about the comment about Zephyr L2 fix.